### PR TITLE
Newreactcomponent

### PR DESCRIPTION
**Pull request by folusho-adeyemi**
## Description

ADDing the react component for the edituser information page and profile page

## Milestones
Milestone 2

## Test Plan
<div style="position: relative; padding-bottom: 62.5%; height: 0;"><iframe src="https://www.loom.com/embed/c94d1133b5264ea69301a12283c71081?sid=72b58a01-8fbd-4c3d-a2c8-12bdd800cbda" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>

<div>
    <a href="https://www.loom.com/share/c94d1133b5264ea69301a12283c71081">
      <p>FashionConnect - 14 July 2023 - Watch Video</p>
    </a>
    <a href="https://www.loom.com/share/c94d1133b5264ea69301a12283c71081">
      
    </a>
  </div>

### CODE REVIEW

File: FashionConnect_api/routes/users.js [hunk 1]
```diff
@@ -89,7 +90,8 @@ router.post('/users/profile', async (req, res) => {
       await user.save()
   
       // Return the user data in the response
-      res.json({ message: "User profile uodated succesfully" });
+      res.json({ user:user, message: "User profile uodated succesfully" });
```
Comment (safirshahbaaz) [inline FashionConnect_api/routes/users.js]: nit: updated :)

File: FashionConnect_api/routes/users.js [hunk 2]
```diff
@@ -89,7 +90,8 @@ router.post('/users/profile', async (req, res) => {
...
+      console.log(res.json({user}))
```
Comment (safirshahbaaz) [inline FashionConnect_api/routes/users.js]: Let's make sure to remove the console.log when you put up PRs.

File: FashionConnect_ui/src/components/EditProfile/EditProfile.jsx [hunk 3]
```diff
@@ -1,39 +1,45 @@
 import "./EditProfile.css";
 import React, { useState,useContext } from "react";
 import { Link, useNavigate } from "react-router-dom";
-
+import { UserContext } from '../../UserContext.jsx';
+// import { response } from "express";
```
Comment (safirshahbaaz) [inline FashionConnect_ui/src/components/EditProfile/EditProfile.jsx:5]: Let's also make sure to remove any commented out code before you put up PRs.

Comment (safirshahbaaz) [reply FashionConnect_ui/src/components/EditProfile/EditProfile.jsx:5]: My process before I usually put up PRs is I go through every file I've changes and I compare what the previous file was and what changes I've made. Then I remove any console.log statements or commented out code. I also make any changes to indents etc. Once I've done all this clean up, I proceed to make a PR.

File: FashionConnect_ui/src/components/EditProfile/EditProfile.jsx [hunk 4]
```diff
@@ -1,39 +1,45 @@
...
 
 
-    const navigate = useNavigate();
+    
     const handleProfileEdit = async (e) => {
       e.preventDefault();
 
       try {
-        const result = await fetch(`http://localhost:3000/users/profile`,{
+        const info = await fetch(`http://localhost:3000/users/profile`,{
```
Comment (safirshahbaaz) [inline FashionConnect_ui/src/components/EditProfile/EditProfile.jsx:21]: Let's also try to use clearer variable naming. This one can be update_profile_info for example.

**Review by safirshahbaaz — APPROVED** [review]
Overall looks good! Let's make sure to address the suggestions I provided in future PRs. :) Good job getting this feature out!!

File: FashionConnect_api/routes/users.js (hunk 2 above)
Comment (folusho-adeyemi) [reply FashionConnect_api/routes/users.js]: Yes  i will henceforth.

File: FashionConnect_ui/src/components/EditProfile/EditProfile.jsx (hunk 3 above)
Comment (folusho-adeyemi) [reply FashionConnect_ui/src/components/EditProfile/EditProfile.jsx:5]: Thanks for this, i will start doing this as from now.

**Review by safirshahbaaz — APPROVED** [review]

**Pull request merged**